.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
           python --version
           pytest --version
       - run: |
-          sudo apt-get install cpanminus
+          sudo apt-get install cpanminus html-xml-utils
```

Comment:
I am not sure about that. shouldnt be https://open.spotify.com/track/3Le4N9DFC3fZQM7jS4gO6F?si=180845d991ae4b60 enough?

Reply:
I guess that's not the URL you meant to paste.

Reply:
no I hope you like it tho :)

Reply:
We discussed this in the unblock meeting. The line `sudo apt-get install cpanminus` is meant to *only* install cpanm because we call this in the next line. The line was never meant to provide proper test dependencies not handled by cpanm. Anyway, you guys overruled me so let's accept it as is :)